test/integration/targets/vmware_inventory/aliases

```diff
@@ -0,0 +1,4 @@
+posix/ci/cloud/group1/vcenter
```

Comment:
The new group alias is `shippable/vcenter/group1`

Reply:
OK.